### PR TITLE
Update pipeline to release when we tag to github and documentation for release

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- Add pipeline to release the package to github
- Add documentation about the release

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,45 @@ This plugin allows for the creation and management of Mattermost kubernetes inst
...
+To release a new version of the `cloud plugin` please do the following steps:
+
+- Bump the version to the desire version in the files:
+  - `plugin.json`
+  - `webapp/src/manifest.js`
+  - `server/manifest.go`
+
+- Create a PR with the changes, see [example](https://github.com/mattermost/mattermost-plugin-cloud/pull/52)
+
+- After the PR is merged, generate the release notes. Here will show the commands but if need more information please refer to the [internal docs](https://app.gitbook.com/@mattermost-private/s/internal-documentation/cloud/cloud/releases/rel-notes)
```
Comment (angeloskyratzakos) [inline README.md:50]:
```suggestion
- After the PR is merged, generate the release notes with the commands below. For more information please refer to the [internal docs.](https://app.gitbook.com/@mattermost-private/s/internal-documentation/cloud/cloud/releases/rel-notes)
```

File: README.md [hunk 2]
```diff
@@ -34,3 +34,45 @@ This plugin allows for the creation and management of Mattermost kubernetes inst
...
 8. Enable the plugin
 The plugin is now usable. Make changes to your plugin and run `make deploy` again to run those changes on your local Mattermost server.
+
+
+## Release
+
+To release a new version of the `cloud plugin` please do the following steps:
+
+- Bump the version to the desire version in the files:
+  - `plugin.json`
```
Comment (jwilander) [inline README.md:44]: You only need to change it here then run `make apply`

Comment (cpanato) [reply README.md]: done

**Pull request merged**